athena/models/speech_transformer.py

```diff
@@ -123,6 +128,9 @@ def __init__(self, data_descriptions, config=None):
         # some temp function
         self.random_num = tf.random_uniform_initializer(0, 1)
 
+        self.deploy_encoder = None
```

Comment:
We should also create a 'encoder+decoder' pb. Creating deploy demo will be easier with a single pb.